app/templates/components/event-card.hbs

```diff
@@ -1,28 +1,61 @@
-<div class="ui fluid event card">
```

Comment:
There's a lot of code-duplication here. please share the common code and add/remove snippets and classes using if statement